src/home/GetGoing.tsx

```diff
@@ -28,20 +23,20 @@ export default factory(function GetGoing({ middleware: { theme, intersection } }
 					classes={{ 'dojo.io/Card': { root: [themedCss.commands], content: [themedCss.commandsContent] } }}
 				>
 					<div classes={[themedCss.command]}>
-						<span classes={[themedCss.commandOne, play ? themedCss.commandOneAnimation : null]}>
+						<span classes={[themedCss.commandOne, themedCss.commandOneAnimation]}>
 							npm i @dojo/cli @dojo/cli-create-app -g
 						</span>
-						<span classes={[play ? themedCss.blinkOne : null]}>|</span>
+						<span classes={[themedCss.blinkOne]}>|</span>
 					</div>
 					<div classes={[themedCss.command]}>
-						<span classes={[themedCss.commandTwo, play ? themedCss.commandTwoAnimation : null]}>
+						<span classes={[themedCss.commandTwo, themedCss.commandTwoAnimation]}>
 							dojo create app --name hello-world
 						</span>
-						<span classes={[play ? themedCss.blinkTwo : null]}>|</span>
+						<span classes={[themedCss.blinkTwo]}>|</span>
 					</div>
 				</Card>
 				<div classes={[themedCss.codeContainer]}>
-					<Card dark classes={{ 'dojo.io/Card': { root: [play ? themedCss.code : themedCss.hide] } }}>
```

Comment:
Should probably remove the `.hide` class completely from the css too.

Reply:
@KaneFreeman good catch, done!